core/src/components/datetime/datetime.ios.scss

```diff
@@ -85,27 +85,37 @@
    */
   @include padding($datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5, $datetime-ios-padding * 0.5);
 
+  align-items: center;
+
   height: calc(100% - #{$datetime-ios-padding});
 }
 
-:host .calendar-day {
-  font-size: 20px;
+:host .calendar-day-wrapper {
```

Comment:
~ nit: Is there a reason for the `:host` selector? I see for `.calendar-day` for example we use both approaches.

Reply:
I was actually wondering the same because I thought it wasn't needed but I see it used in a lot of places so I assumed it had a purpose. Maybe someone else can chime in here.